addons/mail/models/mail_channel.py

```diff
@@ -259,6 +252,14 @@ def write(self, vals):
     def get_alias_model_name(self, vals):
         return vals.get('alias_model', 'mail.channel')
 
+    def get_alias_values(self):
+        """ Return values to create an alias, or to write on the alias after its
+            creation.
+        """
+        values = super(Channel, self).get_alias_values()
+        values.update({"alias_force_thread_id": self.id})
```

Comment:
Shouldn't we set alias_model_id is we remove the alias_model_name context key at create ?

Reply:
In the `create` method of the alias the alias_model_name is placed in the context before create the related mail.alias ; Then the alias_model_id will be set correctly.